src/main/java/org/fcrepo/repository/FedoraResourceImport.java

```diff
@@ -45,21 +46,44 @@
      */
     public static void main(final String[] args) {
 
-        final String fcrepoUrl = System.getProperty("fcrepo.url", "http://localhost:8080/rest/");
-        final String resourcesDir = System.getProperty("resources.dir", ".");
+        String fcrepoUrl = System.getProperty("fcrepo.url", "http://localhost:8080/fcrepo/rest/");
+        if (!fcrepoUrl.endsWith("/")) {
+            fcrepoUrl = fcrepoUrl + "/";
+        }
+        final String resourcesDir = System.getProperty("resources.dir", "./src/main/resources/data");
```

Comment:
No, initially I was going to add the resources using @peichman-umd's Resource Putter, but it seems to traverses the file system to its deepest point and then back. So a directory structure like
```
/dark_archive
              /sunshine
```
has the resource ```/dark_archive/sunshine``` created before ```/dark_archive``` and I would get an error on these types of resources. 

I didn't spend too much time working on this, it is possible that it could be fixed up to use the ```FedoraResourceImport``` for everything. But I was just trying to get it working for now.